### ISSUE TITLE
{Linked:Bug9928798; Linked:Bug1557313; Linked:Bug9652646} [Adaptive Cards> Designer]: When we zoom the page to 400%, Content inside the calendar picker got truncated and is not clearly visible. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8185]
### Target Platforms

Other

### SDK Version

1.3

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25262,1000)
Browser Version: Version 110.0.1556.0 (Official build) dev (64-bit)

**Pre -Requisite:**
1. Set the Display Resolution to '1280 * 1024' and check if scale to 100% and apply the changes.

**Repro Steps:**
1. Open the above URL in edge browser.
2. Navigate to the Designer tab item and press enter key.
3. Navigate to the 'Card Payload Editor' button and press enter key to expand it.
4. Paste the attached Json file into it.
5. Navigate to the 'Preview Mode' and press enter key.
6. Now, zoom the browser to 400%.
7. Navigate to the 'Transaction From Date' date picker button and press enter key and observe the issue.

**Actual Impact:**
On zooming the page at 400%, content under date picker is getting truncated and not visible properly.

**Expected Result:**
Elements under date picker should be visible properly and should not cropped while resizing the page at 400%.

**User Impact:**
People with low vision will find it difficult to navigate through the page if data is truncated or not visible clearly at 400% zoom.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B2ECDA461-6B98-45C8-8FB1-A9399A72DA17%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![MAS1 4 10- Normal mode](https://user-images.githubusercontent.com/95913748/209323056-6bf5d629-b835-4931-9f85-6cd4553d2049.png)
![MAS1 4 10- Elements under Date picker are getting cropped at 400%](https://user-images.githubusercontent.com/95913748/209323069-b790cd10-bc4f-4df7-8a31-8d321aa36f3a.png)

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": 2,
          "items": [
            {
              "type": "Input.Number",
              "id": "InvoiceAmount",
              "label": "Total amount of expenses to be billed",
              "wrap": true,
              "value": 20
            },
            {
              "type": "Input.ChoiceSet",
              "id": "Milestones",
              "style": "expanded",
              "isMultiSelect": true,
              "isRequired": true,
              "errorMessage": "The milestone field is required.",
              "label": "Milestones (Description - Amount(Currency))",
              "wrap": true,
              "choices": [
                {
                  "title": "Invoice1 - 100 (USD)",
                  "value": "Invoice1"
                },
                {
                  "title": "Invoice2 - 135 (USD)",
                  "value": "Invoice2"
                }
              ],
              "value": "Invoice1"
            },
            {
              "type": "Input.Date",
              "id": "FromDate",
              "label": "Transaction from date",
              "value": "2022-12-01"
            },
            {
              "type": "Input.Date",
              "id": "ToDate",
              "label": "Transaction to date",
              "value": "2022-12-08"
            },
            {
              "type": "Input.Text",
              "id": "Comments",
              "isMultiline": true,
              "label": "Comments",
              "wrap": true,
              "value": "test"
            }
          ]
        }
      ]
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "Submit",
      "data": {
        "id": "InvoiceDocument_Submit"
      }
    }
  ]
}
```

**Comment by vagpt** [comment]
This issue can track using below URL.

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9652646  
https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9928798

**Comment by vagpt** [comment]
This issue is working fine at our end.

Please find the below snippet for the reference.

**Snippet:**

![image](https://user-images.githubusercontent.com/95913748/235123311-ffe09460-d8ff-4ae1-9b09-48005fddaef1.png)

**Comment by vaishali1397** [comment]
Issue is repro'ing in below test environment, hence reactivating the bug.
**Test Environment:**
OS Version: 22H2 (Build 25357.1)
Browser: Edge Dev (Version 115.0.1843.0 (Official build) dev (64-bit))

![image](https://github.com/microsoft/AdaptiveCards/assets/104483005/0f4354f9-3d9c-4931-9732-6300b860cc81)

**Comment by jwoo-msft** [comment]
@anna-dingler 

Tossing in your way since you're handling the related issues.
